.gitignore

```diff
@@ -131,3 +131,6 @@ dmypy.json
 # Pycharm
 .idea/
 
+# Binary files for testing
+/examples/data
```

Comment:
shouldn't this be `examples/data`, `examples/screenshots` ?

Reply:
I think either will work, will change to simplified